emcc.py

```diff
@@ -2771,14 +2771,19 @@ def modularize():
       # after document.currentScript is gone, so we save it.
       # (when MODULARIZE_INSTANCE, an instance is created
       # immediately anyhow, like in non-modularize mode)
+      if shared.Settings.EXPORT_ES6:
+        GETSCRIPT = "import.meta.url"
+      else:
+        GETSCRIPT = "typeof document !== 'undefined' && document.currentScript ? document.currentScript.src : undefined"
```

Comment:
No need to make this local variable uppercase.

Would this be better names "script_url" or something like that?

Why is the js var called `_scripteDir`?   Isn't it the full URL?

Reply:
Yeah good point. I didn't name that, but it is a bit confusing. `_scriptDir` can be a blob, or path to a script. Later on `_scriptDir` is assigned to `scriptDirectory`, which contains the directory of the script. I'm not sure what `scriptDirectory` is used for. 
Anyway `import.meta.url` seems to return the same thing as `document.currentScript.src`, the difference being that `import.meta.url` works in a module, while `document.currentScript.src` doesn't.